glossary.asciidoc

```diff
@@ -270,7 +270,7 @@ Require::
 
 Revert::
     Use revert() when you need to handle the same type of situations as <<require-sentence, require()>> but with more complex logic.
-    For instances, if your code have some nested if/else logic flow, you will find that it makes sense to use <<require-sentence, require()>> instead of require().
+    For instance, if your code has some nested if/else logic flow, you will find that it makes sense to use revert() instead of <<require-sentence, require()>>.
```

Comment:
<<var>> is the format for an asciidoc reference/link. Replace it with quotes, please. This will not format predictably. This section needs an actual definition of revert. "The revert function in Solidity is a ..."